src/js/osweb/backends/keyboard.js

```diff
@@ -162,10 +162,14 @@ export default class Keyboard {
    * @return {Array} - List of default values for the given responses.
    */
   _get_default_from_synoniem (responses) {
-    const defaults = []
+    let defaults = []
+    let synonyms
     for (let i = 0; i < responses.length; i++) {
-      const synoniem = this._synonyms(responses[i])
-      defaults.push(synoniem[0])
+      synonyms = this._synonyms(responses[i])

```

Comment:
I don't see synonyms declared anywhere (with let or const), or does that happen somewhere outside of the diff?